doc/releases/changelog-dev.md

```diff
@@ -43,6 +43,9 @@
 
 <h3>Improvements 🛠</h3>
 
+* Added the decomposition of zyz for special unitaries with multiple control wires.
+  [(#6042)](https://github.com/PennyLaneAI/pennylane/pull/6042)
```

Comment:
```suggestion
  [(#6046)](https://github.com/PennyLaneAI/pennylane/pull/6046)
```

Reply:
@trbromley why? PR 6046 is not related to this PR :)